idea/src/META-INF/plugin.xml

```diff
@@ -2988,6 +2988,15 @@ The Kotlin plugin provides language support in IntelliJ IDEA and Android Studio.
                      language="kotlin"
     />
 
+    <localInspection implementationClass="org.jetbrains.kotlin.idea.inspections.SetterBackingFieldAssignmentInspection"
+                     displayName="Setter backing field should be assigned"
```

Comment:
I'd change name to something like "Existing backing field is not assigned by the setter" or "Setter for a property with backing field should assign backing field". Choose what you think is better, or suggest your own name (current name is grammatically incorrect).